src/cloud/docker/docker-extend.md

```diff
@@ -75,6 +75,76 @@ To add a new service version using a `Dockerfile`:
 
 1. Once the build succeeds, test the changes by specifying the [Docker build sources].
 
+## Add a new PHP extension
+
+In addition to built in extensions, you can add PHP extensions to the PHP container.
+
+{:.procedure}
+To add a new PHP extension:
+
+1. Clone the `{{site.data.var.mcd}}` project to your local environment if necessary.
+
+1. On the command line, navigate to the following directory
+   ```bash
+      cd magento-cloud-docker/src/Compose/Php
+   ```
```

Comment:
Some formatting problems with the step and example.

```suggestion
1. On the command line, navigate to the PHP directory.

   ```bash
   cd magento-cloud-docker/src/Compose/Php
   ```
```